azext_edge/edge/_help.py

```diff
@@ -780,6 +780,701 @@ def load_iotops_help():
+    """
+
+    helps[
+        "iot ops dataflow endpoint create fabric-realtime"
+    ] = """
+        type: command
+        short-summary: Create or replace a Microsoft Fabric Real-Time Intelligence data flow endpoint.
+        long-summary: For more information on Microsoft Fabric Real-Time Intelligence dataflow endpoint, see https://aka.ms/aio-fabric-real-time.
+
+        examples:
+        - name: Create or replace a dataflow endpoint resource with minimum input.
+          text: >
+            az iot ops dataflow endpoint create fabric-realtime
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --host "fabricrealtime.servicebus.windows.net:9093"
+        - name: Create or replace a dataflow endpoint resource using SASL authentication method.
+          text: >
+            az iot ops dataflow endpoint create fabric-realtime
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --host "fabricrealtime.servicebus.windows.net:9093"
+            --sasl-type ScramSha256
+            --secret-name mysecret
+        - name: Show config for creating a dataflow endpoint resource.
+          text: >
+            az iot ops dataflow endpoint create fabric-realtime
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --host "fabricrealtime.servicebus.windows.net:9093"
+            --acks One
+            --compression Gzip
+            --group-id mygroupid
+            --partition-strategy Static
+            --max-bytes 200000
+            --cloud-event-attribute CreateOrRemap
+            --disable-tls
+            --show-config
+    """
+
+    helps[
+        "iot ops dataflow endpoint create local-mqtt"
+    ] = """
+        type: command
+        short-summary: Create or replace a Azure IoT Operations Local MQTT dataflow endpoint.
+        long-summary: For more information on Azure IoT Operations Local MQTT dataflow endpoint, see https://aka.ms/local-mqtt-broker.
+
+        examples:
+        - name: Create or replace a dataflow endpoint resource with minimum input.
+          text: >
+            az iot ops dataflow endpoint create local-mqtt
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --hostname aio-broker
+            --port 1883
+        - name: Create or replace a dataflow endpoint resource using X509 authentication method.
+          text: >
+            az iot ops dataflow endpoint create local-mqtt
+            --name myendpoint
+            --instance mycluster-ops-instance
+            --resource-group myresourcegroup
+            --hostname aio-broker
+            --port 1883
+            --secret-name mysecret
+        - name: Create or replace a dataflow endpoint resource with anonymous auth.
```

Comment:
Is "anonymous auth" an official term? "...with no auth." or "...with no auth configured." can work and align with the param (comment applies to other uses of anonymous auth).